ui/helpers/constants/zendesk-url.js

```diff
@@ -1,4 +1,6 @@
 const ZENDESK_URLS = {
+  ACCOUNT_UPGRADE:
```

Comment:
Does this URL need to be updated? It looks identical to the one below it.

Reply:
This is intentional as the page doesn't exist yet, but I didn't want to introduce a broken link.

I'll add a comment to clarify.